Don't override the MACOSX_DEPLOYMENT_TARGET set by MacPorts, and
use the right C++ standard library.
--- setup.py.orig	2019-11-03 20:39:06.000000000 -0600
+++ setup.py	2020-06-18 17:19:39.000000000 -0500
@@ -14,13 +14,13 @@
 
 from setuptools import Extension, setup
 
-MACOS_FLAG = ['-mmacosx-version-min=10.7']
+MACOS_FLAG = []
 FLAGS_POSIX = [
     '-fPIC', '-std=gnu++0x', '-Wall', '-Wno-parentheses', '-Werror=switch',
 ]
-FLAGS_CLANG = ['-c', '-O3'] + FLAGS_POSIX + ['-stdlib=libc++']
-LFLAGS_POSIX = ['-fPIC',  '-lstdc++']
-LFLAGS_CLANG = ['-fPIC', '-stdlib=libc++']
+FLAGS_CLANG = ['-c'] + FLAGS_POSIX + ['-stdlib=@CXX_STDLIB@']
+LFLAGS_POSIX = ['-fPIC']
+LFLAGS_CLANG = ['-fPIC', '-stdlib=@CXX_STDLIB@']
 
 sources = ['pysass.cpp']
 headers = []
